endpoint.go

```diff
@@ -637,6 +639,14 @@ func EndpointOptionGeneric(generic map[string]interface{}) EndpointOption {
 	}
 }
 
+// CreateOptionIpam function returns an option setter for the ipam configuration for this endpoint
+func CreateOptionIpam(prefAddress net.IP, ipamOptions map[string]string) EndpointOption {
```

Comment:
@jc-m @rmb938 
The multiple IPs per interface will not make it to docker 1.10.

I am ok for this PR to pass only one preferred IP per endpoint, and change it to a list in future should we decide to allow the multiple IPs. 
Because it would imply changing an internal interface which is invoked by docker. Not a contract between libnetwork and remote ipam driver.